website/docs/r/team.html.markdown

```diff
@@ -58,6 +58,7 @@ The `organization_access` block supports:
 * `manage_teams` - (Optional) Allow members to create, update, and delete teams.
 * `manage_organization_access` - (Optional) Allow members to update the organization access settings of teams.
 * `access_secret_teams` - (Optional) Allow members access to secret teams up to the level of permissions granted by their team permissions setting.
+* `manage_agent_pools` - (Optional) Allows members to create, edit, and delete agent pools within their organization.
```

Comment:
Non-Blocking Nitpick: I just noticed that the rest of the docs have "Allow" in its singular form, rather than its plural form. Should we update this "Allow" to be consistent with the rest?:
```suggestion
* `manage_agent_pools` - (Optional) Allow members to create, edit, and delete agent pools within their organization.
```